src/components/boards/BoardItem.vue

```diff
@@ -33,10 +33,11 @@
 			{{ board.title }}
 		</div>
 		<div class="board-list-avatars-cell">
-			<Avatar :user="board.owner.uid" class="board-list-avatar" />
+			<Avatar :user="board.owner.uid" :title="board.owner.displayname" class="board-list-avatar" />
```

Comment:
I think the tooltip here is coming from the parent element and the Avatar has a proper v-tooltip usage. My suggestion would be to just set an empty title attribute on the parent div (`.board-list-avatars-cell`)